Makefile

```diff
@@ -14,6 +14,16 @@ ETCDIR_OCID ?= ${ETCDIR}/ocid
 BUILDTAGS := selinux seccomp $(shell hack/btrfs_tag.sh) $(shell hack/libdm_tag.sh)
 BASHINSTALLDIR=${PREFIX}/share/bash-completion/completions
 
+# If GOPATH not specified, use one in the local directory
+ifeq ($(GOPATH),)
+export GOPATH := $(CURDIR)/_output/local/go
```

Comment:
What about using `_output` as the GOPATH? In other words, what's the benefit of having additional `local` and `go` subdirs?